src/app/components/details/details_model.rs

```diff
@@ -176,6 +197,29 @@ impl PlaylistModel for DetailsModel {
         self.state().playback.current_song_id()
     }
 
+    fn playlist_song_ids(&self) -> Option<Vec<String>> {
+        if let Some(album) = self.get_album_description() {
+            let playlist_ids = album
+                .songs
+                .songs
+                .iter()
+                .map(|song| song.id.clone())
+                .collect::<Vec<_>>();
+            return Some(playlist_ids);
+        }
+        None
+    }
+
+    fn playlist_is_playing(&self) -> bool {
+        let current_song_id = self.state().playback.current_song_id();
```

Comment:
that's one way to do it -- but it would be easier/more reliable to look at playback.current_source